projects/core/src/occ/base-occ.module.ts

```diff
@@ -27,6 +29,10 @@ export class BaseOccModule {
           useExisting: WithCredentialsInterceptor,
           multi: true,
         },
+        {
+          provide: OCC_USER_ID_CONSTANTS,
+          useValue: OCC_USER_IDS,
+        },
```

Comment:
Another option would be to provide the token as we instantiate it like so:
```
export const OCC_USER_ID_CONSTANTS = new InjectionToken('OCC_USER_ID_CONSTANTS', {
  providedIn: 'root',
  factory() {
    return OCC_USER_IDS;
  }
});
```

Got this idea from @armanozak and I thought it might be more concise to go with this.  What do you think?

Reply:
Sure, I'll implement that. I didn't even know it accepted a second argument.